static/js/vue-cdr-access/src/components/header/headerHome.vue

```diff
@@ -59,6 +59,7 @@
                 </div>
             </div>
         </div>
+      <meta name="google-site-verification" content="WO17zKdoDATg4FxnQafjjRNowoM-TDP4hEfb_H--mdo" />
```

Comment:
I think we may actually want to that this line to this file in the "head" section with the other meta tags:
https://github.com/UNC-Libraries/box-c/blob/main/static/js/vue-cdr-access/index.html